cimloader/databases/__init__.py

```diff
@@ -14,13 +14,17 @@ class Parameter:
 @dataclass
 class ConnectionParameters:
```

Comment:
This class needs to be renamed or cimgraph.databases.ConnectionParameters needs to be used instead.